circulation-test/circulation_test/blueprints/home.py

```diff
@@ -53,7 +57,7 @@ def _get_loans(access_token):
 def _get_books():
     books = []
     response = requests.get(GROUPS_URL)
-    group_feed = StringIO(response.content)
+    group_feed = BytesIO(response.content)
```

Comment:
How has this changed to Bytes from String?

Reply:
When you access `response.content` you get bytes. On Python 2 that didn't matter but on Python 3 because of the difference in str vs bytes this would error out since StringIO would expect `str` and not `bytes` type.